integrations/sonarqube/.port/resources/port-app-config.yaml

```diff
@@ -1,12 +1,9 @@
 createMissingRelatedEntities: true
 deleteDependentEntities: true
 resources:
-  - kind: projects
+  - kind: ga_projects
     selector:
       query: 'true'
-      apiFilters:
-        filter:
-          qualifier: TRK
```

Comment:
why we remove TRK?

Reply:
The default value is already `TRK` so there isn't any need to pass it.